sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -587,6 +587,7 @@ public static <K, V> Read<K, V> read() {
         .setCommitOffsetsInFinalizeEnabled(false)
         .setDynamicRead(false)
         .setTimestampPolicyFactory(TimestampPolicyFactory.withProcessingTime())
+        .setConsumerPollingTimeout(Duration.standardSeconds(1L))
```

Comment:
Should the default be increased slightly? Maybe 2 seconds? Given that this is even causing issues in the US to US cross-regional reads, it does seem to suggest that 1 second might be a bit too aggressive as a general default.

Reply:
This makes sense. @xianhualiu please update https://github.com/apache/beam/blob/master/CHANGES.md as well about your fix, since this should be called out in our release notes.

Reply:
increased to 2 seconds